docs/guides/deploy-guide/manager.md

```diff
@@ -43,7 +43,16 @@ on the seed node. The deployment of the seed node itself is documented in the
 cd environments/manager
 ```
 
-### Step 1: Create operator user
+### Step 1: Configure deployment branch.
```

Comment:
I would prefer to not have this as a separate step. It is sufficient to have main there by default and it will just work. This can be added somewhere else.

Reply:
I automatically used a different branch because my repo is configured to force a review of changes. Presumably this will also be the case in many productive environments, e.g. for compliance reasons from the outset. 
I have now marked the step as optional and added a clear note.

Reply:
It's still confusing for newcomers. I would prefer to have a section in the configuration guide about some more advanced use cases, like working with branched configuration repositories.

Reply:
Done, moved to https://github.com/osism/osism.github.io/blob/manager_install_and_bootstrap/docs/guides/configuration-guide/manager.md#working-with-git-branches
Added a reference link to the Deployguide Page.